layouts/partials/html-head.hbs

```diff
@@ -18,4 +18,7 @@
     <link rel="alternate" href="/{{site.locale}}/tsc-minutes.xml" title="Node.js TSC meeting minutes" type="application/rss+xml">
     <link rel="stylesheet" href="/{{site.locale}}/styles.css" media="all">
     <link rel="stylesheet" href="/static/css/prism-tomorrow.css" media="all">
+
+    <script src="//use.typekit.net/mse5tqx.js"></script>
+    <script>try{Typekit.load();}catch(e){}</script>
```

Comment:
We only use that font in one place on the front page which is why it's in a CSS include to google fonts on the front page. I don't know what the benefit of typekit is but if we do include it for this font we should only do so on the front page.